docs/pages/docs/faq.mdx

```diff
@@ -52,6 +52,174 @@ defined and incompatible interfaces - which are the biggest problems in common
 specifications.
 </FAQBox>
 
+<FAQBox title="How does Quint compare to TLA+?">
+
+Quint is based on TLA+ and uses the same underlying logic (Temporal Logic of
+Actions), so it is very similar to TLA+ in terms of where it is useful. The main
+differences between Quint and TLA+ are:
+1. Quint has programming-style syntax, while TLA+ uses math/latex symbols.
+2. Quint has more static analysis, such as type checking and different modes
+   (`pure def`, `action`, etc), while TLA+ has no native support for types nor
+   explicit distinction between the modes (or levels, as they are called in
+   TLA+'s documentation).
+3. Quint tooling is more similar to what we have for programming languages:
+   near-instant feedback as you type the code, CLI-first executable, an easy way
+   to obtain one execution of your spec in the terminal, a REPL, located error
+   messages, etc. These are all not currently available in TLA+, or available
+   with significant constraints.
+4. Quint won't let you write many things that are allowed in TLA+. TLA+ has a
+   proof system (TLAPS), while Quint doesn't. Therefore, it is possible to write
+   more advanced formulas in TLA+ than in Quint, even ones that are not
+   supported by the existing model checkers, and then prove them using the proof
+   system. Quint restricts you to a fragment of TLA with the intention of
```

Comment:
Yeah, it was on purpose. Both make sense, right. I just figure it is easier to explain to people that both Quint and TLA+ are based on TLA (although both add ZFC on top of TLA, so there is actually more that is shared).

Reply:
I think saying TLA+ in this sentence would be incorrect. Quint is not based on TLA+ the spec language, but on TLA the logic.

Reply:
In this sense, you are correct. But also, I've seen TLA+ described as:

> The “+” in TLA+ uses a formal set theory based on ZFC to allow TLA variables to take on many kinds of values (atoms like numbers and strings, finite and infinite sequences, sets, records, and functions).

in some places before (I think Lamport wrote this). And since Quint also has ZFC tihngs, I think it is also right to say it is a fragment of TLA+

Reply:
That's right. Probably it's better to say "Quint restricts you to a fragment of the underlying logic of TLA+". The underlying logic of TLA+ is TLA plus a variant of ZFC. The "+" also adds non-logical constructs such as modules.